### DISCUSSION TITLE
How to make the vim.inspect work?

### DISCUSSION THREAD

**Discussion by OriLiMu** [LazyVim/LazyVim#1993] [Q&A] [answered]
I add code in plugin which help me debug something but `vim.inspect` not work here. print has a notify popout
```lua
    vim.inspect(args)
    -- print(args)
```

**Comment by dpetka2001** [comment, marked as answer]
`vim.inspect` expects an object as an argument, like a table for example. You can use `vim.print` which prints both objects and non-objects args.
